code/datums/emotes.dm

```diff
@@ -0,0 +1,353 @@
+	id = "wince_heavy"
+	message_targeted = "#SOURCE winces painfully at #TARGET!"
+	message_untargeted = "#SOURCE winces painfully!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/blink
+	id = "blink"
+	message_targeted = "#SOURCE blinks at #TARGET."
+	message_untargeted = "#SOURCE blinks."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/blink_rapid
+	id = "blink_rapid"
+	message_targeted = "#SOURCE rapidly blinks at #TARGET."
+	message_untargeted = "#SOURCE rapidly blinks."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/cough
+	id = "cough"
+	message_targeted = "#SOURCE coughs on #TARGET."
+	message_untargeted = "#SOURCE coughs."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_DISTANCE
+
+/datum/emote/sneeze
+	id = "sneeze"
+	message_targeted = "#SOURCE sneezes on #TARGET."
+	message_untargeted = "#SOURCE sneezes."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_DISTANCE
+
+/datum/emote/sniff
+	id = "sniff"
+	message_targeted = "#SOURCE sniffs #TARGET."
+	message_untargeted = "#SOURCE sniffs."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_DISTANCE
+
+/datum/emote/heave
+	id = "heave"
+	message_targeted = "#SOURCE dry heaves!"
+	message_untargeted = "#SOURCE dry heaves!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/sputter
+	id = "sputter"
+	message_targeted = "#SOURCE sputters!"
+	message_untargeted = "#SOURCE sputters!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/moan
+	id = "moan"
+	message_targeted = "#SOURCE moans."
+	message_untargeted = "#SOURCE moans."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/moan_heavy
+	id = "moan_heavy"
+	message_targeted = "#SOURCE moans in pain!"
+	message_untargeted = "#SOURCE moans in pain!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/shiver
+	id = "shiver"
+	message_targeted = "#SOURCE shivers at #TARGET."
+	message_untargeted = "#SOURCE shivers."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/wail
+	id = "wail"
+	message_targeted = "#SOURCE wails at #TARGET!"
+	message_untargeted = "#SOURCE wails!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/hiss
+	id = "hiss"
+	message_targeted = "#SOURCE hisses at #TARGET."
+	message_untargeted = "#SOURCE hisses."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/mrowl
+	id = "mrowl"
+	message_targeted = "#SOURCE mrowls at #TARGET."
+	message_untargeted = "#SOURCE mrowls."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/red
+	id = "red"
+	message_targeted = "#SOURCE goes red."
+	message_untargeted = "#SOURCE goes red."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/scratch
+	id = "scratch"
+	message_targeted = "#SOURCE scratches #TARGET.."
+	message_untargeted = "#SOURCE scratches themself."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/jump
+	id = "jump"
+	message_targeted = "#SOURCE jumps at #TARGET!"
+	message_untargeted = "#SOURCE jumps!"
```

Comment:
The targeted and non-targeted versions of these emotes have very different connotations.